modules/rte_mis_mail/src/Form/EmailSettingsForm.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Drupal\rte_mis_mail\Form;
+
+use Doctrine\Common\Annotations\Annotation\Required;
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\mobile_number\MobileNumberUtilInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides a form to configure Mobile Number settings.
+ */
+class EmailSettingsForm extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'custom_email_validation_settings_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return ['custom_email_validate.settings'];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $config = $this->config('custom_email_validate.settings');
+
+    $form['heading'] = [
+        '#type' => 'markup',
+        '#markup' => '<h2>Email Services</h2>',
+    ];
+    
+    $form['verify'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Email Verification'),
+      '#default_value' => $config->get('verify'),
+      '#description' => $this->t('Verification requirement.'),
+    ];
+
+    $form['message'] = [
+      '#type' => 'textarea',
+      '#title' => $this->t('Verification Message'),
+      '#default_value' => $config->get('message'),
+      '#description' => $this->t('The email to send during verification. Replacement parameters for verification code is !code.'),
+      '#states'=>['visible' => [
+        ':input[name="verify"]' => 
+          ['checked' => TRUE],
+        ],
+      ]
+    ];
+    $form['#cache']['tags'] = ['config_evf.settings'];
```

Comment:
Why we have added this cache tag?

Reply:
Forgot to remove this...was trying something.